guiqwt/plot.py

```diff
@@ -1301,7 +1301,7 @@ def adjust_ycsw_height(self, height=None):
 
     def xcsw_is_visible(self, state):
         if state:
-            QApplication.processEvents()
+            #QApplication.processEvents()

```

Comment:
```suggestion

```
Just remove the line instead of commenting it.

Reply:
ok thanks, Truth be told I have been running for months with the first proposed change but opted for what I thought would be the less intrusive option 2 of commenting line 1304 but I just ran into a segfault, unless someone has a better suggestion I am am going to revert back to connecting the SIG_VISIBILITY_CHANGED signal for this PR